components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -1528,10 +1540,10 @@ public void update(HttpServletRequest request) {
     }
 
     /**
-     * @param wstrustEp
+     * @param wstrustEps
      */
-    public void setWstrustEp(String wstrustEp) {
```

Comment:
It's not good practice to remove the public methods. Shall we deprecate this and introduce the new one?

Reply:
Addressed in my last commit. Kept both methods since the base type of the attribute changed from String to List<String>